src/components/ChatBox/Incoming.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { Image } from 'react-bootstrap'
+const Incoming = () => {
```

Comment:
This component should receive a `prop` that is the message. Then render this `prop`